barretenberg/ts/src/main.ts

```diff
@@ -100,8 +105,10 @@ async function initClientIVC(bytecodePath: string, crsPath: string) {
   const api = await Barretenberg.new({ threads });
 
   debug('loading BN254 and Grumpkin crs...');
+  // TODO(https://github.com/AztecProtocol/barretenberg/issues/1097): tighter bound needed
+  // currently using 1.6x points in CRS because of structured polys, see notes for how to minimize
   // Plus 1 needed! (Move +1 into Crs?)
-  const crs = await Crs.new(2 ** 18 + 1, crsPath);
+  const crs = await Crs.new(2 ** 19 + 1, crsPath);
```

Comment:
why the bump?

Reply:
because all of the SRS' had to be bumped by 1.5x due to Adam's initial PR. Issue here: https://github.com/AztecProtocol/barretenberg/issues/1097